Issue with special characters in output message

Quotes in the output message are not displayed correctly anymore. Might be related to #41 (not sure how though).

Example (see last line)
```
# lasagna [lasagna.test]
./lasagna_test.go:39:14: undefined: RemainingOvenTime
FAIL	lasagna [build failed]
&#39;/usr/local/go/bin/go test --short --json .&#39; returned exit code 2: exit status 2
```

--- a/app/javascript/components/editor/TestRunOutput.tsx
+++ b/app/javascript/components/editor/TestRunOutput.tsx
@@ -14,7 +14,7 @@ export const TestRunOutput = ({
     />
   ) : (
     <pre className="v1-message">
-      <code>{testRun.messageHtml}</code>
+      <code dangerouslySetInnerHTML={{ __html: testRun.messageHtml }} />
     </pre>
   )
 }

--- a/app/javascript/components/editor/TestRunSummary.tsx
+++ b/app/javascript/components/editor/TestRunSummary.tsx
@@ -136,7 +136,7 @@ const TestRunSummaryContent = ({
         <div className="error-message">
           <h3>We received the following error when we ran your code:</h3>
           <pre>
-            <code>{testRun.messageHtml}</code>
+            <code dangerouslySetInnerHTML={{ __html: testRun.messageHtml }} />
           </pre>
         </div>
       )

